houdiniproject/noticeme
# Convert to typescript

- Update to the latest yarn
- Update yarn.lock
- Add typescript and types
- Run tsc --init
- Rename files to .ts
- Ignore generated JS files
- Convert to typescript
- Remove unwanted exec permissions from cmd.ts
- Add yarn build and clean
- Update NOTICE
- Update version
